### ISSUE TITLE
GPU hang when moving around in Panorama

### ISSUE THREAD

**Issue by glubsy** [ValveSoftware/Dota-2#1771]
#### Your system information

* System information from steam: in a [gist](https://gist.github.com/glubsy/8930de18e83adb497905097091e9789c/ab5711fd21856585e7312824e940051313ee4ef9)
* Have you checked for system updates?: Yes, fully updated 1 day ago
* Are you using the latest stable video driver available for your system? Yes
* Have you verified the game files?: No

#### Please describe your issue in as much detail as possible:
When moving around in the main menu (Panorama?), sometimes the GPU will hang, the screen will freeze and the system becomes unresponsive to keyboard input.
Logging through SSH, I attempt to kill Dota2 (which is still running) and Steam client but to no avail. See system log [here](https://gist.github.com/glubsy/60742a68138642a96600fc620f30af5f).
Relevant lines are
```
21:47:41 kernel: NVRM: GPU at PCI:0000:01:00: GPU-3d2e1720-3ac2-b3d0-73cf-e44dab47add3                                                                                                           
21:47:41 kernel: NVRM: Xid (PCI:0000:01:00): 31, pid=1070, Ch 00000034, intr 10000000. MMU Fault: ENGINE CE2 HUBCLIENT_CE2 faulted @ 0x1_1db46000. Fault is of type FAULT_PTE ACCESS_TYPE_WRITE
21:47:48 kernel: nvidia-modeset: ERROR: GPU:0: Error while waiting for GPU progress: 0x0000917c:0 2:0:3624:3616
```
This is by using Vulkan renderer, **update:** the OpenGL renderer doesn't cause any problem.

#### Steps for reproducing this issue:
Difficult to reproduce consistently but it mostly consists of
1. Moving around in the menus (Battle Pass, player profile, guild profile, match end-report...)
2. Play a game (it seems to help having at least one end match-report in cache)
3. While a game match is running, move *back* and *forth* in the Panorama history (by clicking on the small arrows in the top left corner) to get to a previously viewed page
4. Frozen

I have reported the issue on the nVidia forum [here](https://forums.developer.nvidia.com/t/gpu-hang-in-dota2-leads-to-system-crash/125552).

**Comment by glubsy** [comment]
Unfortunately I crashed as soon as I installed the [440.66.15 driver](https://aur.archlinux.org/packages/nvidia-vulkan/).
I was in the hero selection menu, and right after clicking "Back to Dashboard", the GPU crashed after displaying the main menu. 
I have disabled the vulkan renderer to see if this issue occurs there too. I'll make a new post in the Vulkan repo if this doesn't crash. So far it seems much more stable. Going back to dashboard doesn't produce the usual 1 second freeze like I experience with the vulkan renderer.
